internal/tools/wallet.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/onflow/flow-cli/pkg/flowkit/output"
 	"github.com/onflow/flow-cli/pkg/flowkit/services"
 
-	devWallet "github.com/onflow/fcl-dev-wallet"
+	devWallet "github.com/onflow/fcl-dev-wallet/go/wallet"
```

Comment:
you need to use `go get` command to update the dev-wallet dependency so this linter doesn't complain and there are go.sum entries.

Reply:
We should also release a tagged version of dev-wallet to do this update.

Reply:
The module import in go.mod didn't change, it's still github.com/onflow/fcl-dev-wallet
I think the linter complains because it uses a tagged version of dev wallet that doesn't have the corresponding changes yet. 
I am coordinating the release of dev wallet so once it's done this error should not happen